### ISSUE TITLE
Is this project even active anymore?

### ISSUE THREAD

**Issue by JemiloII** [jindw/xmldom#246]
For seeing millions of downloads a week, but many PRs and Issues open, it makes me believe that it isn't actively maintained.

**Comment by amitguptagwl** [comment]
No! it is not. You can try [fast-xml-parser](https://github.com/NaturalIntelligence/fast-xml-parser/issues/134) instead

**Comment by karfau** [comment]
See #259
